PyKCS11/__init__.py

```diff
@@ -1406,6 +1488,23 @@ def generateKeyPair(self, templatePub, templatePriv,
             raise PyKCS11Error(rv)
         return ck_pub_handle, ck_prv_handle
 
+    def deriveKey(self, template, baseKey, mecha=MechanismAESGENERATEKEY):
```

Comment:
Why do you add deriveKey() here if you remove it later in https://github.com/LudovicRousseau/PyKCS11/pull/93/commits/c11249ed47f81c75ec8d9a8e2d3f6a25ebcf1e82 ?

Reply:
I merged WIP PR#72 and PR#91 having conflicts resulting in two instances of deriveKey. I sorted that in another commit.

Reply:
derivekey() has 2 tests. Only one works with softhsm2. I have successfully tested both with another provider.

Reply:
I should not see that in the final PR.
Please fix it.

Reply:
I only see one deriveKey() when comparing all commits with master.